build.gradle

```diff
@@ -36,4 +38,28 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
+apiValidation {
+    ignoredPackages += [
+            'io/customer/base/extenstions',
+            'io/customer/sdk/api',
+            'io/customer/sdk/data/store',
+            'io/customer/sdk/queue',
+            'io/customer/sdk/extensions',
+            'io/customer/sdk/util',
+    ]
+
+    ignoredProjects += [
+            'common-test',
+            'app',
```

Comment:
The purpose of this PR is to help us prevent adding breaking changes to the SDK by accident. And purposely introduce public features to the SDK. 

Because of that, I believe that the only gradle modules in the project that need to be included in the validator are public facing modules: `sdk` and `messaging-push`. All others can be ignored.